zaza/charm_lifecycle/prepare.py

```diff
@@ -118,4 +120,5 @@ def main():
     if not isinstance(level, int):
         raise ValueError('Invalid log level: "{}"'.format(args.loglevel))
     logging.basicConfig(level=level)
+    print('model_name: {}'.format(args.model_name))
```

Comment:
Was this left in in error?

Reply:
To be able to use the auto-generated name in a script (ref #137) it would need to be printed.

The `juju models` output that Zaza prints later will potentially contain many zaza-models so that is not directly usable.

I guess we could hide it behind a verbose flag or something, or do you have other suggestions / preferences?

Reply:
I guess I can also only make it print when the auto-generate functionality is used (i.e. -m is not specified on command line)

Reply:
I think its fine, I just missed the point.